diff --git a/services/frontend/src/layout/HomeContainer.jsx b/services/frontend/src/layout/HomeContainer.jsx
index 49812d2b4..58ec10df3 100644
--- a/services/frontend/src/layout/HomeContainer.jsx
+++ b/services/frontend/src/layout/HomeContainer.jsx
@@ -44,11 +44,6 @@ export default function HomeContainer () {
     label: 'Deployments',
     iconName: 'RocketIcon',
     disabled: !enableSidebarFirstLevel
-  }, {
-    link: '/activities',
-    label: 'Activities',
-    iconName: 'CheckListIcon',
-    disabled: !enableSidebarFirstLevel
   }]
 
   const [topItems, setTopItems] = useState([])
