diff --git a/services/frontend/src/Router.jsx b/services/frontend/src/Router.jsx
index b40046f62..74a1062d1 100644
--- a/services/frontend/src/Router.jsx
+++ b/services/frontend/src/Router.jsx
@@ -7,8 +7,7 @@ import AllApplications from './components/applications/all/AllApplications'
 import RecommendationsHistory from './components/recommendations/RecommendationHistory'
 import Settings from './components/settings/Settings'
 import Profile from './components/profile/Profile'
-import { getApiActivities, getApiActivitiesUsers, getApiDeploymentsHistory, getApiApplication, getApiActivitiesTypes, getApplicationsRaw, getApiPod } from './api'
-import Activities from '~/components/application/activities/Activities'
+import { getApiDeploymentsHistory, getApiApplication, getApplicationsRaw, getApiPod } from './api'
 import DeploymentHistory from '~/components/application/deployment_history/DeploymentHistory'
 import AppDetails from '~/components/application/detail/AppDetails'
 import ApplicationContainer from '~/layout/ApplicationContainer'
@@ -18,7 +17,6 @@ import AutoscalerPodDetailContainer from '~/layout/AutoscalerPodDetailContainer'
 import Taxonomy from './components/taxonomy/Taxonomy'
 import Caching from './components/caching/Caching'
 import AllDeployments from './components/deployments/AllDeployments'
-import AllActivities from './components/activities/Activities'
 import Services from './components/application/services/Services'
 import ErrorPage from './pages/ErrorPage'
 
@@ -179,15 +177,6 @@ export function getRouter () {
             return { totalCount, deployments: d, applications }
           },
           element: <AllDeployments />
-        },
-        {
-          loader: async () => {
-            const applications = await getApplicationsRaw()
-            return { applications }
-          },
-          id: 'activities',
-          path: '/activities',
-          element: <AllActivities />
         }
       ]
     },
@@ -214,25 +203,6 @@ export function getRouter () {
           id: 'application/details',
           element: <AppDetails />
         },
-        {
-          path: 'activities',
-          id: 'application/activities',
-          loader: async (loaderObject) => {
-            const { applicationId } = loaderObject.params
-            const [activities, users, types] = await Promise.all([
-              getApiActivities(applicationId),
-              getApiActivitiesUsers(),
-              getApiActivitiesTypes()
-            ])
-            return {
-              applicationId,
-              activities,
-              users,
-              types
-            }
-          },
-          element: <Activities />
-        },
         {
           path: 'deployment-history',
           id: 'application/deploymentHistory',
